libs/sdk-ui-dashboard/src/model/state/dateFilterConfig/dateFilterConfigConverters.ts

```diff
@@ -0,0 +1,142 @@
+// (C) 2019-2021 GoodData Corporation
```

Comment:
i suspect that this kind of code would better fit into _staging? there i wanted to concentrate stuff that may part of some models/code that would belong to other packages. for example this.. looks like we can benefit of having it in the sdk-ui-filters?

Reply:
Moved there